Podcast/SeriesCollectionViewCell.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  SeriesCollectionViewCell.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 11/16/16.
+//  Copyright © 2016 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class SeriesCollectionViewCell: UICollectionViewCell {
+    
+    var seriesImageView: UIImageView!
+    
+    var seriesImage: UIImage? {
+        didSet {
+            if let seriesImage = seriesImage {
+                seriesImageView.image = seriesImage
+            }
+        }
+    }
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        backgroundColor = UIColor.white
+        
+        seriesImageView = UIImageView(frame: CGRect(x: 0, y: 0, width: frame.size.width, height: frame.size.height))
```

Comment:
don't need `size` for swift 3 anymore :) `frame.width` and `frame.height` should do